app/lib/identity_tijuana/member_sync.rb

```diff
@@ -78,6 +147,88 @@ def self.import_user(user_id, sync_id)
       calling_subscription: ['associated_audits', 'Member_subscription', %w[unsubscribed_at]],
     }.freeze
 
+    # Find the primary TJ user match for a given ID member.
+    #
+    # Looks at the user for each given id, orders them by created at
+    # date, then selects:
+    #
+    #  1. The user with a normalised (but not cleaned), matching email, if any
+    #  2. The first subscribed user, if any
+    #  3. The first remaining user
+    def self.find_primary_user_for_member(member)
+      external_ids = member.member_external_ids.where(system: 'tijuana')
+      if external_ids.nil? || external_ids.count == 0
+        raise "Id member has no linked TJ users"
+      end
+
```

Comment:
Should we first check `external_ids.count == 1`, and then just return it.
I understand that we return `return users.first` at the end of this call, but if there is only a single TJ user linked, then we don't need to do any other processing.